pages/common/yadm-bootstrap.md

```diff
@@ -0,0 +1,9 @@
+# yadm-bootstrap
+
+> Execute Yadm's bootstrap file.
```

Comment:
```suggestion
> Execute `yadm`'s bootstrap file.
```
You sometimes type Yadm, sometimes yadm, and sometimes `yadm`. It should be the same in all the places.